$OpenBSD: patch-source4_heimdal_lib_roken_version-script_map,v 1.1 2021/05/12 23:33:24 jca Exp $

Don't add a version to the symbols in this library, it breaks binutils-2.17 ld(1).

Index: source4/heimdal/lib/roken/version-script.map
--- source4/heimdal/lib/roken/version-script.map.orig
+++ source4/heimdal/lib/roken/version-script.map
@@ -1,4 +1,4 @@
-HEIMDAL_ROKEN_1.0 {
+{
 	global:
 		arg_printusage;
 		arg_printusage_i18n;
